application/src/main/java/org/togetherjava/tjbot/features/moderation/TransferQuestionCommand.java

```diff
@@ -92,12 +96,18 @@ public void onMessageContext(MessageContextInteractionEvent event) {
         String originalChannelId = event.getTarget().getChannel().getId();
         String authorId = event.getTarget().getAuthor().getId();
         String mostCommonTag = tags.get(0);
+        String chatGptPrompt = String.format(
+                "Generate topic for question: %s. Minimum %s and maximum %s characters.",
```

Comment:
I don't know if it's a good idea to rely on GPT for char limit for title, might be a good idea to check the title length before setting title, if title generated is greater than our limit you can trim the title using `String#substring` instead as backup.

Reply:
I did that because chatgpt was giving long topic names which really wasn't suitable. Trimming that long response generate bad output overall. I have attached test responses for your reference.
![image](https://github.com/Together-Java/TJ-Bot/assets/55508798/efebf07c-0593-4e99-913c-80120e9cafdd)
![image](https://github.com/Together-Java/TJ-Bot/assets/55508798/21d2793a-7624-41ad-8ef9-d7a621a81360)

Reply:
I have addressed rest of your comments

Reply:
can we use both? Like request gpt to generate shorter responses and check if it's actually within limit, if yes then go ahead otherwise we should trim it.

Reply:
Yes that can be done